spell/spell-reviewer-goerli-checklist.md

```diff
@@ -178,16 +178,16 @@ Spell Actions (Per Exec Sheet):
         * [ ] IF combining `val` of multiple RWA ilks being combined, `val` calculation is done once per ilk and added to make the total, with workings provided in code comments. The existing `val` value can be retrieved by calling `read()` on `PIP_RWAXX` and converting the result into decimal.
     * [ ] Poke `spotter` to pull in the new price
 * [ ] Payments
-* [ ] Payments are not crafted or reviewed on Goerli (comments may be present)
+  * [ ] Payments are not crafted or reviewed on Goerli (comments may be present)
 * [ ] Debt Ceiling (`line`) + Liquidation Oracle Price Bump (`val`)
```

Comment:
On the second thought this seems to be a subsection related to the `RWA`, so needs to be moved up and be part of the other top-level section, right?

Reply:
Seems like so. To me makes most sense as well.